src/app/complex/complex-results/complex-results.component.html

```diff
@@ -1,41 +1,77 @@
 <div class="margin-top-large margin-bottom-large row">
   <ng-container *ngIf="complexSearch;else loadingSpinner">
-  <div class="columns medium-12" *ngIf="complexSearch.totalNumberOfResults !== 0">
-    <h2 class="padding-left-large">Total number of results: {{complexSearch.totalNumberOfResults}} </h2>
-    <div class="columns medium-4">
-      <cp-complex-filter *ngIf="spicesFilter && bioRoleFilter && interactorTypeFilter"
-                         [spicesFilter]="spicesFilter"
-                         [bioRoleFilter]="bioRoleFilter"
-                         [interactorTypeFilter]="interactorTypeFilter" [facets]="complexSearch.facets"
-                         (onResetAllFilters)="onResetAllFilters()"
-                         (onSpicesFilterChanged)="onSpicesFilterChanged($event)"
-                         (onBiologicalRoleFilterChanged)="onBiologicalRoleFilterChanged($event)"
-                         (onInteractorTyoeFilterChanged)="onInteractorTyoeFilterChanged($event)">
-      </cp-complex-filter>
+    <div class="columns medium-12" *ngIf="complexSearch.totalNumberOfResults !== 0">
+      <h2 class="padding-left-large">Total number of results: {{ complexSearch.totalNumberOfResults }} </h2>
+      <div class="columns medium-4">
+        <cp-complex-filter *ngIf="spicesFilter && bioRoleFilter && interactorTypeFilter"
+                           [spicesFilter]="spicesFilter"
+                           [bioRoleFilter]="bioRoleFilter"
+                           [interactorTypeFilter]="interactorTypeFilter" [facets]="complexSearch.facets"
+                           (onResetAllFilters)="onResetAllFilters()"
+                           (onSpicesFilterChanged)="onSpicesFilterChanged($event)"
+                           (onBiologicalRoleFilterChanged)="onBiologicalRoleFilterChanged($event)"
+                           (onInteractorTypeFilterChanged)="onInteractorTypeFilterChanged($event)">
+        </cp-complex-filter>
+      </div>
+      <div class="columns medium-8">
+        <cp-complex-paginator [currentPageIndex]="currentPageIndex"
+                              [lastPageIndex]="lastPageIndex"
+                              (onPageChange)="onPageChange($event)"></cp-complex-paginator>
+        <div class="listOfResults" *ngIf="DisplayType==false">
+          <div class="displayButton" style="text-align: center">
+            <input type="submit" name="submit" value="Complex navigator" class="button"
+                   (click)="toggleDisplayType()">
+          </div>
+          <cp-complex-list [complexSearch]="complexSearch">
+          </cp-complex-list>
+        </div>
+        <div class="Complex-navigator" *ngIf="DisplayType">
```

Comment:
The same way we have a component `cp-complex-list`, we should have a component for the complex navigator with the sorting and the table all in that component, and here we simply do
```
<cp-complex-navigator [complexSearch]="complexSearch"
                                  [interactors]="allInteractorsInComplexSearch"
                                  [interactorsSorting]="interactorsSorting">
            </cp-complex-navigator>
```
I guess one issue with that is that we have the toggle display button with the sorting buttons, but I think they should remain separate. We should have one (or even 2, depending on how we do it) toggle button above to change the view, and then, after that, the sorting buttons.

Reply:
Then we can move all the logic and html about the sorting to the `cp-complex-navigator` component.

Reply:
Then it would look like this 

https://github.com/Complex-Portal/complex-portal-view/assets/114929382/2860832b-360e-48a5-83b3-55bc4c0e0470